index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+  <title>GintsMisins Portfolio</title>
+  <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+  <header class="title-row">
+    <h4 class="header_content" id="logo">GintsMisins</h4>
+    <p class="header_content" id="nav">≡</p>
+  </header>
```

Comment:
- [ ] Please ensure you use the font family used in the design for this section which is Poppins as shown in the design.

Reply:
Thanks for the advice but the linter showed some issues with family font, so I went to check for that inhttps://stylelint.io/user-guide/rules/list/font-family-no-missing-generic-family-keyword/ , and took one of mentioned font
 maybe I didn't get it right
I will try to change it back.